frontend/public/components/resource-list.tsx

```diff
@@ -22,18 +24,22 @@ export const ResourceListPage = connectToPlural((props: ResourceListPageProps) =
     return <ErrorPage404 />;
   }
 
+  const showGettingStarted = flags.OPENSHIFT && !flags.PROJECTS_AVAILABLE && kindObj.labelPlural !== 'Projects';
```

Comment:
Need to distinguish between `undefined` and `false` for `PROJECTS_AVAILABLE`. Otherwise the message could flash when we're still getting the projects. See #208